manifests/service.pp

```diff
@@ -4,8 +4,6 @@
 #
 class telegraf::service {
 
-  assert_private()
```

Comment:
can you explain why you removed this?

Reply:
Sorry, that was actually a facepalm moment. Just removed that.